servers/display_server.cpp

```diff
@@ -636,8 +638,8 @@ void DisplayServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_window_list"), &DisplayServer::get_window_list);
 	ClassDB::bind_method(D_METHOD("get_window_at_screen_position", "position"), &DisplayServer::get_window_at_screen_position);
 
-	ClassDB::bind_method(D_METHOD("create_sub_window", "mode", "vsync_mode", "flags", "rect"), &DisplayServer::create_sub_window, DEFVAL(Rect2i()));
-	ClassDB::bind_method(D_METHOD("delete_sub_window", "window_id"), &DisplayServer::delete_sub_window);
+	ClassDB::bind_method(D_METHOD("_create_sub_window", "mode", "vsync_mode", "flags", "rect"), &DisplayServer::create_sub_window, DEFVAL(Rect2i()));
+	ClassDB::bind_method(D_METHOD("_delete_sub_window", "window_id"), &DisplayServer::delete_sub_window);
```

Comment:
Do they need to be bound at all? Usually we'd only bind them hidden if they're needed for UndoRedo calls by StringName, but this doesn't seem to be the case here since you're not change any other occurrence.

Reply:
Probably not, unless someone wants to implement fully custom versions of `Viewport` and `Window`.

Reply:
I'd suggest removing them for now then, and we can reassess if someone comes up with a use case for exposing them. Otherwise even if they're hidden, users could still use them and we'd have to give support for that use case.